cli/sub_command.go

```diff
@@ -309,6 +316,53 @@ func (c *subCmd) validateInputs(nc *nats.Conn) error {
 		}
 	}
 
+	if c.jetStream {
+		mgr, err := jsm.New(nc)
```

Comment:
should use the existing helpers to get a mgr because those will set up tracing and stuff for you